src/docs/doxygen/Doxyfile.in

```diff
@@ -228,7 +228,7 @@ TAB_SIZE               = 4
 # "Side Effects:". You can put \n's in the value part of an alias to insert
 # newlines.
 
-ALIASES                =
+ALIASES                = "accel=\xrefitem accelerated_list \"\"\"\"" 
```

Comment:
Suggestion:  The expanded string `\accelerated` might be clearer to developers that are only looking at the raw doxygen in the `hpp` files than `\accel`.

Reply:
Changed.  This change spilled over to several header files.